### PR TITLE
add runnable parameters convertor

### PR DESCRIPTION
**Pull request by Vartanchik**
Add possibility to set request parameters in behat test like runnable string.
Main goal is to get actual date time when tests are running.

### CODE REVIEW

File: src/Context/ApiContext.php [hunk 1]
```diff
@@ -299,6 +301,60 @@ protected function compareStructureResponse(string $variableFields, PyStringNode
         }
     }
 
+    /**
+     * @param array $requestParams
+     *
+     * @When I have :requestParams with runnable values (placed in <>)
+     *
+     * @Then :requestParams should be converted with executing runnable code
```
Comment (alekseytupichenkov) [inline src/Context/ApiContext.php]: Is it just to describe the method or does it do something?

File: src/Context/ApiContext.php [hunk 2]
```diff
@@ -299,6 +301,60 @@ protected function compareStructureResponse(string $variableFields, PyStringNode
...
+            if (is_array($value)) {
+                $requestParams[$key] = $this->convertRunnableCodeParams($value);
+                continue;
+            }
+
+            if (!is_string($value)) {
+                continue;
+            }
+
+            $pregMatchValue = preg_match('/^<.*>$/', $value);
```
Comment (alekseytupichenkov) [inline src/Context/ApiContext.php]: Maybe also add `trim($value)` ?

**Review by alekseytupichenkov — CHANGES_REQUESTED** [review]
And please remove all `.DS_Store` files

PS: Also, I recommend adding those lines to `~/.gitignore` file to prevent adding unnecessary files to git repositories
```
.idea
.DS_Store
```

File: src/Context/ApiContext.php (hunk 1 above)
Comment (Vartanchik) [reply src/Context/ApiContext.php]: Just to describe. Already removed it.

File: src/Context/ApiContext.php (hunk 2 above)
Comment (Vartanchik) [reply src/Context/ApiContext.php]: Good point.

File: src/Context/ApiContext.php [hunk 3]
```diff
@@ -299,6 +301,53 @@ protected function compareStructureResponse(string $variableFields, PyStringNode
...
+                continue;
+            }
+
+            $pregMatchValue = preg_match('/^<.*>$/', trim($value));
+
+            if ($pregMatchValue === 0 || $pregMatchValue === false) {
+                continue;
+            }
+
+            $command = substr($value, 1, -1);
```
Comment (alekseytupichenkov) [inline src/Context/ApiContext.php]:
```suggestion
            $command = substr(trim($value), 1, -1);
```

**Review by alekseytupichenkov — CHANGES_REQUESTED** [review]

**Review by alekseytupichenkov — APPROVED** [review]

**Comment by alekseytupichenkov** [comment]
It will be amazing 👍

**Review by Yozhef — APPROVED** [review]

**Pull request merged**